DOC-2313-Added a NOTE (#255)
tigergraph/gsql-docs: modules/querying/pages/select-statement/select-statement-v1.adoc

====

== SAMPLE

The SAMPLE clause is an optional clause that selects a uniform random sample from the population of edges or target vertices specified in the FROM argument.

[NOTE]
====
* In *V1* syntax, the `SAMPLE` clause supports left-to-right directed edges and undirected edges.

> * In *V2* and *V3* syntax, only left-to-right directed edges are supported. Right-to-left directed edges (e.g., `FROM start:s \<-(:e)- :v`) are not supported in either V2 or V3 syntax.

Reviewer: ```suggestion
* In *V2* and *V3* syntax, the `SAMPLE` clause supports left-to-right directed and undirected edges. Right-to-left directed edges (e.g., `FROM start:s \<-(:e)- :v`) are not supported in either V2 or V3 syntax.
```

> * The `SAMPLE` clause does not guarantee returning the exact number of elements; the number of samples returned may vary.

Reviewer: ```suggestion
* The `SAMPLE` clause does not guarantee returning the exact number of elements; it may return fewer than the requested number.
```
